package.json

```diff
@@ -25,7 +25,8 @@
     "postinstall": "node scripts/npm/postinstall",
     "release": "./scripts/travis/release.sh",
     "build": "gulp build",
-    "test": "gulp test && node test/accessibility/wcag.js"
+    "test": "gulp test && node test/accessibility/wcag.js",
+    "cf-link": "cd ./src/cf-pagination && npm link"
```

Comment:
We can chain the rest of the packages as they are completed. Arguments are pointless since you could just use npm link if you wanted to manually go through each component. When this release is finished, we can remove this command.